arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -66,5 +66,18 @@ public async Task<TruncateCollectionResult> TruncateCollectionAsync(string colle
                 throw await GetApiErrorException(response);
             }
         }
+
+        public async Task<GetCollectionCountResponse> GetCollectionCountAsync(GetCollectionCountOptions options = null)
```

Comment:
`options` shouldn't be `null` here. We need a mandatory parameter for the route.